README.md

```diff
@@ -62,6 +64,40 @@ Then call any method available in the `ExceptionHandling::Methods` mixin:
       flash.now['error'] = "A specific error occurred. Support has been notified."
     end
 
+### Tagging Exceptions in Honeybadger
+
+⚠️ Honeybadger differentiates tags by spaces and commas, so you should **not** include spaces or commas in your tags.
+
+⚠️ Tags are case-sensitive.
+
+#### Manually Tagging Exceptions
+
+Add `:honeybadger_tags` to your `log_context` usage with an array of strings.
+
+```ruby
+log_error(ex, "A specific error occurred.", honeybadger_tags: ["critical", "sequoia"])
+```
+
+**Note**: Manual tags will be merged with any automatic tags.
+
+#### Automatically Tagging Exceptions (`honeybadger_auto_tagger=`)
+
+Configure exception handling so that you can automatically apply multiple tags to exceptions sent to honeybadger.
+
+The Proc must accept an `exception` argument that will be the exception in question and must always return an array of strings (the array can be empty).
+
+Example to enable auto-tagging:
+```ruby
+ExceptionHandling.honeybadger_auto_tagger = ->(exception) do
+  exception.message =~ /fire/ ? ["high-urgency", ""] : ["low-urgency"]
```

Comment:
Nit: `=~` has fallen a out of favor these days because it returns a whole `MatchData` object. I've read that best practice is `.match?(/fire/)` which doesn't pay that cost. (Too bad, because `=~` is more succinct.)